test/performance/odp_atomic_perf.c

```diff
@@ -620,7 +620,75 @@ static inline void test_atomic_xchg_u64(void *val, void *out, uint32_t num_round
 	*result = ret;
 }
 
-static inline void test_atomic_load_acq_u32(void *val, void *out, uint32_t num_round)
+static inline void test_atomic_bit_set_u32(void *val, void *out ODP_UNUSED, uint32_t num_round)
+{
+	odp_atomic_u32_t *atomic_val = val;
+
+	for (uint32_t i = 0; i < num_round; i++)
+		odp_atomic_bit_set_u32(atomic_val, 0x1);
+}
+
+static inline void test_atomic_bit_fetch_set_u32(void *val, void *out ODP_UNUSED,
+						 uint32_t num_round)
+{
+	odp_atomic_u32_t *atomic_val = val;
+
+	for (uint32_t i = 0; i < num_round; i++)
+		(void)odp_atomic_bit_fetch_set_u32(atomic_val, 0x1);
```

Comment:
Fetch type API calls should use the return value for something, so that compiler will not change the atomic instruction to a store type version of it.

See previous tests for fetch type calls:

`
for (uint32_t i = 0; i < num_round; i++)
		ret += odp_atomic_fetch_dec_u64(atomic_val);
	*result = ret;
`